### ISSUE TITLE
Only some Emojis appear when using ttf-emojione package in Arch Linux

### ISSUE THREAD

**Issue by DoronCohen** [emojione/emojione-assets#41]
I haven't go to the reason why but only some of the emojis are displayed right on my Linux machine. This is both on my browser and in terminal. How can I debug this? I saw @maximbaz  is the maintainer of this and understood that this is the right place for this issue.

[Screenshot of browser](https://ibb.co/N11f6GL)

**Comment by maximbaz** [comment]
AUR page is the right place to ask questions for Arch Linux package 😉 

Have you run this command as per the pinned comment?

```
ln -sf ../conf.avail/75-emojione.conf /etc/fonts/conf.d/75-emojione.conf
```

**Comment by DoronCohen** [comment]
Thanks! That worked perfectly. I will just note that it will default to view numbers as emojis when viewing websites with no configured font e.g. github.

**Comment by maximbaz** [comment]
Yep, but that's the issue of those websites, not the font, if you are interested I can point you to an example of the implementation that does it correctly.

**Comment by DoronCohen** [comment]
Interested. Although I solved it by defining a default font in my browser

**Comment by maximbaz** [comment]
The trick is to define an `emoji` font, but only allow it to render a specific set of codepoints, those that are defined as Emoji as per Unicode standard. This way it's guaranteed that emojis are rendered with the `emoji` font, and everything else is rendered as text.

For example, [here's how I made it for Wire app.](https://github.com/wireapp/wire-webapp/blob/7c492905ccfa7f019274e1d0cfa9ec2ed0cf0065/src/style/common/variables.less#L20-L94)
